### ISSUE TITLE
snaRSS doesn't handle 302 redirects

### ISSUE THREAD

**Issue by blackwind** [fullphat/snarl#8]
As we've previously discussed, snaRSS doesn't handle 302 redirects. Because some feeds always redirect no matter what, simply procuring the final URL isn't always an option. For such feeds, snaRSS will need to support redirection.

If it helps, a simple PHP test script might read as follows:

    <?
    if (!$_GET["test"]) {
      header(" ", TRUE, 302);
      header("Location: ".$_SERVER["PHP_SELF"]."?test=1");
    } else {
      echo "Test!";
    }
    ?>

**Comment by fullphat** [comment]
I agree.  snaRSS is now SANDSTORM (https://fullphat.net/?p=382) however this still doesn't support 302s.  I'm considering recoding SANDSTORM in C#.
